src/SFA.DAS.ManageApprenticeships/Application/Queries/Transfers/GetFinancialBreakdown/GetFinancialBreakdownHandler.cs

```diff
@@ -23,25 +24,39 @@ public GetFinancialBreakdownHandler(IForecastingApiClient<ForecastingApiConfigur
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow));
 
-            await Task.WhenAll(transferFinancialBreakdownTask, pledgesTask);
+            var nextYearBreakdowntask = await _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow.AddYears(1)));
 
+            var yearAfterNextYearBreakdowntask = await _forecastingApiClient.Get<GetTransferFinancialBreakdownResponse>
+                                                        (new GetTransferFinancialBreakdownRequest(request.AccountId, DateTime.UtcNow.AddYears(2)));
+            var pledgesTask = await _levyTransferMatchingApiClient.Get<GetPledgesResponse>
+                                                        (new GetPledgesRequest(request.AccountId));
             return new GetFinancialBreakdownResult
             {
-                Commitments = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsOut.Commitments),
-                ApprovedPledgeApplications = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsOut.ApprovedPledgeApplications),
-                TransferConnections = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsOut.TransferConnections),
-                AcceptedPledgeApplications = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsOut.AcceptedPledgeApplications),
-                PledgeOriginatedCommitments = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsOut.PledgeOriginatedCommitments),
-                FundsIn = transferFinancialBreakdownTask.Result.Breakdown.Sum(x => x.FundsIn),
-                ProjectionStartDate = transferFinancialBreakdownTask.Result.ProjectionStartDate,
-                NumberOfMonths = transferFinancialBreakdownTask.Result.NumberOfMonths,
-                AmountPledged = pledgesTask.Result.Pledges.Where(p => p.Status != PledgeStatus.Closed).Sum(x => x.Amount)
+                Commitments = breakdownTask.Breakdown.Sum(x => x.FundsOut.Commitments),
+                ApprovedPledgeApplications = breakdownTask.Breakdown.Sum(x => x.FundsOut.ApprovedPledgeApplications),
+                TransferConnections = breakdownTask.Breakdown.Sum(x => x.FundsOut.TransferConnections),
+                AcceptedPledgeApplications = breakdownTask.Breakdown.Sum(x => x.FundsOut.AcceptedPledgeApplications),
+                PledgeOriginatedCommitments = breakdownTask.Breakdown.Sum(x => x.FundsOut.PledgeOriginatedCommitments),                
+                FundsIn = breakdownTask.Breakdown.Sum(x => x.FundsIn),
+                ProjectionStartDate = breakdownTask.ProjectionStartDate,
+                NumberOfMonths = breakdownTask.NumberOfMonths,
+                CurrentYearEstimatedCommittedSpend = (breakdownTask.Breakdown.Sum(x => x.FundsOut.ApprovedPledgeApplications) +
```

Comment:
As the formula for calculating these values is the same for each financial year, we have a lot of duplicated code here and I can't help but think we could de-dupe it and improve the readability.

Reply:
yes sure, duplication has been removed.